pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/NamespaceStatsAggregator.java

```diff
@@ -70,7 +71,19 @@ public static void generate(PulsarService pulsar, boolean includeTopicMetrics, b
         LongAdder topicsCount = new LongAdder();
         Map<String, Long> localNamespaceTopicCount = new HashMap<>();
 
-        printDefaultBrokerStats(stream, cluster);
+        LongAdder brokerTopicsCount = new LongAdder();
```

Comment:
I suggest following the same convention as aggregating topic level stats to namespace level stats to reduce clutter: Create `AggregatedBrokerStats` and follow same design as `AggregatedNamespaceStats`

Before
```java
if (includeTopicMetrics) {
                    topicsCount.add(1);
                    TopicStats.printTopicStats(stream, topicStats, compactorMXBean, cluster, namespace, name,
                            splitTopicAndPartitionIndexLabel);
                } else {
                    namespaceStats.updateStats(topicStats);
                }
```
add `brokerLevelStats.updateStats(topicStats)`

and before
```java
 if (includeTopicMetrics) {
            printTopicsCountStats(stream, localNamespaceTopicCount, cluster);
        }
```
add `printBrokerLevelMetrics(brokerLevelStats)`

This way you don't need LongAdder or DoubleAdder since you won't have any thread race condition.